apps/condo/domains/banking/hooks/useBankReportTaskUIInterface.tsx

```diff
@@ -68,21 +78,21 @@ export const useBankReportTaskButton = (props: UserBankReportTaskButtonProps) =>
     const intl = useIntl()
     const CreateReportTitle = intl.formatMessage({ id: 'pages.condo.property.report.createReport.title' })
 
-    const { organizationId, user, bankAccount, type = 'primary' } = props
+    const { organizationId, userId, bankAccount, type = 'primary' } = props
 
     const { BankReportTask: TaskUIInterface } = useBankReportTaskUIInterface()
 
-    const { loading, handleRunTask } = useTaskLauncher(TaskUIInterface, {
+    const { loading, handleRunTask } = useTaskLauncher<BankAccountReportTaskCreateInput>(TaskUIInterface, {
         dv: 1,
         sender: getClientSideSenderInfo(),
         account: { connect: { id: bankAccount.id } },
         progress: 0,
         organization: { connect: { id: organizationId } },
-        user: { connect: { id: get(user, 'id') } },
+        user: { connect: { id: userId } },
     })
 
     const BankReportTaskButton = useCallback(() => (
-        <Button type={type} onClick={handleRunTask} loading={loading}>
+        <Button type={type} onClick={() => handleRunTask()} loading={loading}>
```

Comment:
What is the purpose of this change?

Reply:
`onClick` passes data to the handler that we don't expect. 
Sometimes we want to call a handler with some of our own parameters

Reply:
What kind of data? Your change will create new link for onClick handler when render was happened